### DISCUSSION TITLE
I need help, Log = 00: Unk cmd: 0x84

### DISCUSSION THREAD

**Discussion by ComboNeo1981** [darthcloud/BlueRetro#134] [Q&A] [answered]
Gentlemen, I followed a turorial, I did the pinout, controller installation everything correct, but when I turn on the PS2 even pairing the XBox series S/X controller and even the PS3 controller (already changed the bluetooth address) and even managing to enter the page configuration https://blueretro.io/ controls do not respond.
I installed putty to generate .log and the result is this error:
=~=~=~=~=~=~=~=~=~=~=~= PuTTY log 2022.01.07 19:55:05 =~=~=~=~=~=~=~=~= ~=~=~=
# BT_HCI_EVT_LE_ADVERTISING_REPORT
# BT_HCI_EVT_LE_ADVERTISING_REPORT
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
# BT_HCI_EVT_INQUIRY_COMPLETE
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84
00: Unk cmd: 0x84

Any suggestion?

**Comment by darthcloud** [comment]
Any pic of your wiring? External or internal to console?

0x84 is a PS2 memory card cmds, are you plug on the memory card slot??

**Reply by ComboNeo1981** [reply]
Sorry for the delay, I'm using cable for 1 controller only, and it's plugged into the controller's socket, that is, external.

In the tutorial I saw, people only used 5 cables, in another I see that you must connect other cables to close the circuit so as not to detect control 2, is that it?

**Reply by darthcloud** [reply]
5 cable is fine, please take pictures

**Comment by ComboNeo1981** [comment]
Sorry for the delay..
![fios](https://user-images.githubusercontent.com/97318941/149854581-328a270c-c9ea-489c-8b92-c552f006dab9.jpg)
![fios2](https://user-images.githubusercontent.com/97318941/149854585-165eb314-b3af-4c8d-87b4-cf150552f604.jpg)
![fios3](https://user-images.githubusercontent.com/97318941/149854586-92d4406b-08a7-4f7a-bf07-aa4f7960f91b.jpg)
.

**Reply by darthcloud** [reply]
You got no No power connection. If you are powering from USB you still to wire GND to console.

**Comment by ComboNeo1981** [comment, marked as answer]
It worked, I am feeding via USB and I connected the GND of the control cable, it worked as your tip.
The tutorial I followed, didn't have it connected and in the person's video, it worked without GND.
